frontend/src/components/common/icons.tsx

```diff
@@ -35,3 +35,30 @@ export const CloseOutlineIcon = (): React.ReactElement => (
     />
   </Icon>
 );
+
+export const ChevronUpIcon = (): React.ReactElement => (
+  <Icon viewBox="0 0 24 24">
+    <path
+      fill="currentColor"
+      d="M18 15a1 1 0 0 1-.64-.23L12 10.29l-5.37 4.32a1 1 0 0 1-1.41-.15 1 1 0 0 1 .15-1.41l6-4.83a1 1 0 0 1 1.27 0l6 5a1 1 0 0 1 .13 1.41A1 1 0 0 1 18 15z"
+    />
+  </Icon>
+);
+
+export const ChevronDownIcon = (): React.ReactElement => (
+  <Icon viewBox="0 0 24 24">
+    <path
+      fill="currentColor"
+      d="M12 16a1 1 0 0 1-.64-.23l-6-5a1 1 0 1 1 1.28-1.54L12 13.71l5.36-4.32a1 1 0 0 1 1.41.15 1 1 0 0 1-.14 1.46l-6 4.83A1 1 0 0 1 12 16z"
+    />
+  </Icon>
+);
+
+export const BookIcon = (): React.ReactElement => (
```

Comment:
just want to confirm if the book icon is what design wants? i see the default react logo on figma lmao (unless i'm looking at the wrong file) but i think book looks better